plaintext files/labs and verifications.md

```diff
@@ -0,0 +1,109 @@
+1. Create an Issue to add your bio to the repo. The issue should contain each of the following: your name, the information about your from Issue #3, a request to add your bio, and some markdown formatting to make it all look pretty. This way the information is there for whoever ends up fixing the issue (spoiler alert, in this case that's going to be you).  
```

Comment:
@1point618 what is the point of this file compared to the lab text in the files above. Not sure what I should be reviewing here.